### DISCUSSION TITLE
Whitelisting domain on Dashboard

### DISCUSSION THREAD

**Discussion by goeksu** [Web3Auth/Web3Auth#1115] [Help]
Hi. I am trying to publish my app which is successfully authorizing and working on localhost. After I hosted on net, I got error while redirecting.

Here the error I get on console: OpenLogin - RPC Error: could not validate redirect, please whitelist your domain: https://twoedapp.goksu.in for provided clientId q5xWXEpy0sD7JByr9Zl5dc28SU0QykTn at https://dashboard.web3auth.io"

Even if I looked everywhere on Dashboard, I couldnt find anywhere to add my domain on whitelist. You can reach my app on the link below. Thanks..

https://twoedapp.goksu.in

**Comment by amatevosyan-github** [comment]
Hi! Here is how it should work: 
First thing is to create a Plug and play project for testnet. Then you have to create a project for mainnet **using the same project name!**
![image](https://user-images.githubusercontent.com/54032374/209103312-01310042-72f0-41da-8cba-8c739f34a001.png)
Then, inside this new project for mainnet you should add a whiltelisted URL
![image](https://user-images.githubusercontent.com/54032374/209103886-f24baf75-c155-43b7-aa5c-1871df6f9690.png)
Dont forget to use the clientId from your **Mainnet project**

**Reply by goeksu** [reply]
I fixed it myself at the same time you posted. Thanks...
